tests/vxlan/test_vxlan_ecmp.py

```diff
@@ -1970,3 +1970,425 @@ def test_vxlan_varying_src_ip(self, setUp, encap_type):
+                                              tc1_end_point_list[0], "up")
+            time.sleep(10)
+            self.dump_self_info_and_run_ptf("test1", encap_type, True)
+
+            # Single primary backup Failure.
+            # Endpoint list = [A, A`]. Primary[A]| Active  NH=[A`]  A is DOWN  |
+            # Action: A` goes Down | result NH=[]
+            # No active Endpoint results in route being removed.
+            Logger.info("Single primary & backup Failure.")
+            down_list = tc1_end_point_list
+            if isinstance(down_list, str):
+                down_list = [down_list]
+            self.setup['list_of_downed_endpoints'] = set(down_list)
+            # setting both down. no traffic is recieved.
+            ecmp_utils.set_vnet_monitor_state(self.setup['duthost'],
+                                              tc1_new_dest,
+                                              ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)],
+                                              tc1_end_point_list[1], "down")
+            ecmp_utils.set_vnet_monitor_state(self.setup['duthost'],
+                                              tc1_new_dest,
+                                              ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)],
+                                              tc1_end_point_list[0], "down")
+ 
+            time.sleep(10)
+            self.dump_self_info_and_run_ptf("test1", encap_type, True)
+
+        except Exception:
+            ecmp_utils.create_and_apply_priority_config(
+                self.setup['duthost'],
+                vnet,
+                tc1_new_dest,
+                ecmp_utils.HOST_MASK[ecmp_utils.get_payload_version(encap_type)],
+                tc1_end_point_list,
+                [tc1_end_point_list[0]],
+                "DEL")
+
+    def test_vxlan_priority_multi_pri_sec_switchover(self, setUp, encap_type):
+        '''
+            tc2:create tunnel route 1 with 6 endpoints a = {A, B, A`, B`}. A,B
+            are primary, A`,B` are secondary.
+            1) All eps are up.A,B,A`,B`
+            2) send packets to the route 1's prefix dst. packets are received at A,B.
+            3) bring A down.
+            4) send packets to the route 1's prefix dst. packets are received at B.
+            5) bring B down.
+            6) send packets to the route 1's prefix dst. packets are recieved at A`,B`.
+            7) bring B` down.
+            8) send packets to the route 1's prefix dst. packets are recieved at A`.
+            9) bring A up.
+            10) send packets to the route 1's prefix dst. packets are recieved at A.
+            11) bring B,C,C` up.
+            12) send packets to the route 1's prefix dst. packets are recieved at A,B.
+            13) Bring all endpoints down.
+            14) no traffic being passed.
+            15) bring B,C,C` up.
+            16) send packets to the route 1's prefix dst. packets are recieved at A,B.
```

Comment:
line 15 is not bringing up A. Is that a typo?
Also please add one more test to bring standby first and then active after all are down.

Reply:
this was a typo. i have fixed it . added the extra test.